page/src/components/CalendarGrid/CalendarGrid.jsx

```diff
@@ -1,13 +1,11 @@
-import {useMemo, useState} from 'react';
+import {useMemo} from 'react';
 
 import Calendar from 'components/Calendar/Calendar';
 
 import 'styles/CalendarGrid.css';
 
 const CalendarGrid = ({year}) => {
-  const [months, setMonths] = useState([]);
-
-  useMemo(() => {
```

Comment:
This use of `useMemo` + the `useState` was weird, useMemo is for memoizing data, so I changed that.